types/page/page-tests.ts

```diff
@@ -53,41 +90,10 @@ page('/', index2);
 page('/user/:name', load, show);
 page('*', notfound);
 page();
-// everything below is not part of page.js
-// just callbacks etc..
-function text(str: string) {
-    document.querySelector('p').textContent = str;
-}
-function index2() {
-    text('Click a user below to load their avatar');
-    (<HTMLElement>document.querySelector('img'))
-    .style.display = 'none';
-}
-var load: PageJS.Callback = function (ctx, next) {
-    // check if we have .state.avatar already available
-    // this could for example be a cached html fragment.
-    if (ctx.state.avatar) {
-        ctx['avatar'] = ctx.state.avatar;
-        next();
-        return;
-    }
-    // pretend we're querying some database etc
-    setTimeout(function () {
-        // you can assign properties to the context
-        // for use between these functions. The .state
-        // property is what's saved in history.
-        ctx.state.avatar = ctx['avatar'] = avatars[ctx.params.name];
-        ctx.save();
-        next();
-    }, 600);
-}
-var show: PageJS.Callback = function (ctx) {
-    var img: any = document.querySelector('img');
-    img.src = ctx['avatar'];
-    img.style.display = 'block';
-    text('Showing ' + ctx.params.name);
-}
-function notfound() {
-    document.querySelector('p')
-    .textContent = 'not found';
-}
+
+declare var iframe: HTMLIFrameElement;
+const otherPage = page.create({
```

Comment:
can you add an extra line to test that you can call it without options: `page.create()`

Reply:
Done, via PR suggestions tool, thx!